### DISCUSSION TITLE
Home / Top repositories : Remove a project.

### DISCUSSION THREAD

**Discussion by Amis-BASH** [community/community#146949] [Projects and Issues] [answered]
### Select Topic Area

General

### Body

Hello,
I Can see 20 private repo to m'y Top repositories.
I see 1 public project, https://github.com/filegator

I want remove this public project and only see m'y private project.

I want delete this : 
https://github.com/filegator/filegator/issues/504
https://github.com/filegator/filegator/issues/505

I can't delete this, please, a admin for delete this 2 issues ?

I use a second account for post issues.
Thanks you.

**Comment by shunnagahara** [comment, marked as answer]
Issues #504 and #505 in filegator/filegator can only be removed if

- Repository owner
- The GitHub organization administrator
- The person who created the issue (using the account you created it under)

If you are not one of the above, you should contact someone who is.
